packages/mobile/src/transactions/ExchangeFeedItem.tsx

```diff
@@ -186,4 +186,5 @@ const styles = StyleSheet.create({
   },
 })
 
+// @ts-ignore
```

Comment:
Why do we need this?

Reply:
Similar to https://github.com/celo-org/celo-monorepo/pull/325#discussion_r311428027